test/e2e/apps/daemon_set.go

```diff
@@ -174,6 +174,27 @@ var _ = SIGDescribe("Daemon set [Serial]", func() {
 		framework.ExpectNoError(err, "error waiting for daemon pod to revive")
 	})
 
+	ginkgo.It("should list all daemon and delete a collection of daemons with a label selector", func() {
+		label := map[string]string{daemonsetNameLabel: dsName}
+		labelSelector := labels.SelectorFromSet(label).String()
+
+		ginkgo.By(fmt.Sprintf("Creating a DaemonSet %q", dsName))
+		ds, err := c.AppsV1().DaemonSets(ns).Create(context.TODO(), newDaemonSet(dsName, image, label), metav1.CreateOptions{})
+		framework.ExpectNoError(err)
+
+		ginkgo.By("List all Daemonsets")
+		_, err = c.AppsV1().DaemonSets(ns).List(context.TODO(), metav1.ListOptions{})
```

Comment:
How about checking the created daemonset exists on the list which is gotten on the above API?